02-atmos/components/terraform/output-results/main.tf

```diff
@@ -0,0 +1,63 @@
+variable "print_users_weather_enabled" {
+  default     = false
+  type        = bool
+  description = "Whether or not to pretty print the weather results"
+}
+
+data "terraform_remote_state" "weather" {
+  backend = "local"
+
+  config = {
+    path = "${path.root}/../fetch-weather/terraform.tfstate.d/example/terraform.tfstate"
+  }
+}
```

Comment:
In a future tutorial, we should show how to use the `remote-state` module of the `terraform-yaml-stack-config` module